source/sass/wagtail/blocks/listing-block.scss

```diff
@@ -0,0 +1,12 @@
+.listing-block {
+  .mozfest & {
+    .streamfield-content {
```

Comment:
I seem to be missing it. Can you add some context where this class is applied to this block?

Reply:
It gets added in [base_streamfield_block.html](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/integration/mozfest-2024/network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/base_streamfield_block.html#L7) which [listing_block.html](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/integration/mozfest-2024/network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/listing_block.html#L1) extends.